ingestion/src/metadata/ingestion/source/database/datalake.py

```diff
@@ -261,7 +268,7 @@ def get_columns(self, df):
                 else:
                     data_type = "STRING"
```

Comment:
here instead of hardcoding the values like `STRING`  & `INT` we can use `DataType.INT` and  `DataType.STRING`
(ref `metadata.generated.schema.entity.data.table.DataType`)

also We can reduce this if condition to something like 

```
DATALAKE_INT_TYPES = {"int64", "object" } # define this as constant 
.
.
.
data_type = DataType.STRING
if hasattr(df[column], "dtypes") and df[column].dtypes.name in DATALAKE_INT_TYPES:
    data_type = DataType.INT
```

WDYT? 🤔